woodwork/column_accessor.py

```diff
@@ -270,7 +270,7 @@ def _validate_logical_type(self, logical_type):
 
         if isinstance(logical_type, Ordinal):
             logical_type._validate_data(self._series)
-        elif logical_type == LatLong:
+        elif isinstance(logical_type, LatLong):
```

Comment:
I was going to say we should be using `_get_ltype_class` here (and we probably should have before), but it got me thinking about whether we need `_get_ltype_class` anymore. The reason we needed it before was because we needed a way of comparing logical types when some would be instantiated and some would not be, but now that they're always going to be instances, I think we can always use `isinstance` anywhere that we're using `_get_ltype_class`.

Reply:
I'll refactor to remove `_get_ltype_class`

Reply:
Removed `_get_ltype_class` wherever it didn't break too many tests. This results in the behavior where `DataFrame.ww._schema.set_types` will not instantiate logical types.